src/index.js

```diff
@@ -1,19 +1,48 @@
+// eslint-disable-next-line
 import _ from 'lodash';
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+const list = document.getElementById('list');
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
- element.classList.add('hello');
+const toDos = [
+  {
+    description: 'Wash car',
+    completed: false,
+  },
```

Comment:
- [ ] Kindly add an index to each todo.

Reply:
Hi @algerina , Thank you for the review It was great effort. I think you should have tagged @Zeph180.
@Zeph180 Please go through the changes requested.
Thanks.

Reply:
Alright im on it @bhskr44  @algerina